### ISSUE TITLE
Openid2 delegation doesn't work

### ISSUE THREAD

**Issue by greboid** [csmith/Poidsy#21]
Myopenid attempts to authenticate as the pre delegated id rather than the delegated one

**Comment by csmith** [comment]
http://greg.holmes.name/ has this markup:

```
<link rel="openid2.provider" href="http://www.myopenid.com/server"> 
<link rel="openid2.local_id" href="http://greboid.myopenid.com/">    
```

And Poidsy sends a request with:

```
openid.mode=checkid_setup
openid.identity=http://greg.holmes.name/
openid.claimed_id=http://greboid.myopenid.com/
```

myopenid doesn't seem to like the identity value being passed

**Comment by csmith** [comment]
Fix delegate and identity being confused in OpenID 2 requests

Closed by 9f74fa2e59bd4b6939687d824c5a3e75f51a4299
